.gitignore

```diff
@@ -117,3 +117,7 @@ settings.py
 
 # idea files
 .idea/
+
+# cachi2 default files
+cachi2-output/
+cachi2.env
```

Comment:
Should I drop this commit since I saw https://github.com/containerbuildsystem/cachi2/issues/597 ?

Reply:
No, I think those are closely related to cachi2 and it makes sense to keep them in the project scope.

Reply:
Nope, these are our files so while the particular use case is uncommon and kinda a corner case, it's still ours.

Reply:
I would reword the commit message a bit to avoid 1st person narration and replace it with 3rd person narration as is common with technical writings. Also, sentences begin with capital letters, but that's just me nitpicking :).

Reply:
Uhm, looking at the PR, this commit should really be proposed standalone, outside the context of this particular PR.

Reply:
Will this PR be merged actually? Bruno already created a new one https://github.com/containerbuildsystem/cachi2/pull/601

Reply:
Yes. #601 should only merge the basic template. This will be refactored after #601 comes in.

Reply:
> What is currently there is only an example to give us ground for discussion.

Correct, this PR has all the context, so the work should continue here.